src/SubunitPage.js

```diff
@@ -1,43 +1,93 @@
 // SubunitPage.js
 import { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
-import { MantineProvider } from "@mantine/core";
+import { MantineProvider, Button } from "@mantine/core";
 import ModelPage from "./ModelRendering/ModelPage.js";
 import { CharacterCustomizationProvider } from "./ModelRendering/CharacterCustomizationContext.jsx";
-
+import Notes from './NotesFeature/AddingNotesFeature.jsx'
 import './SubunitPage.css';
 import Menu from "./Navigation.js";
+import jsPDF from 'jspdf';
+import { Drawer, Box, IconButton} from '@mui/material';
+import { Close, RingVolumeRounded } from '@mui/icons-material';

```

Comment:
I get a warning that RingVolumeRounded is defined but not used. Can you remove this dependency?